src/sphinxcontrib/towncrier/_towncrier.py

```diff
@@ -1,5 +1,5 @@
 """Towncrier related shims."""
-
+import dataclasses
```

Comment:
This will cause compatibility problems and needs to be dealt with.

Reply:
Do you want to backport dataclasses for python3.6?

Reply:
No, it doesn't make sense for us.